server/src/domain/job/job.service.ts

```diff
@@ -188,6 +188,15 @@ export class JobService {
         break;
 
       case JobName.LINK_LIVE_PHOTOS:
+        // Notify clients to hide the linked live photo asset
```

Comment:
I don't think this is correct. This will run and emit ids for _every_ live motion asset whenever you re-run metadata extraction (for all assets).

Reply:
This stuff should go in `metadata.service.ts` line 150-160-ish instead.

Reply:
Okay that was way easier than I thought it'd be. Updated now. Thank you!